sql/core/v1.2.1/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFilters.scala

```diff
@@ -176,11 +176,11 @@ private[sql] object OrcFilters extends OrcFiltersBase {
 
       case Or(left, right) =>
         for {
-          _ <- createBuilder(dataTypeMap, left, newBuilder, canPartialPushDownConjuncts = false)
-          _ <- createBuilder(dataTypeMap, right, newBuilder, canPartialPushDownConjuncts = false)
+          _ <- createBuilder(dataTypeMap, left, newBuilder, canPartialPushDownConjuncts = true)
+          _ <- createBuilder(dataTypeMap, right, newBuilder, canPartialPushDownConjuncts = true)
```

Comment:
Hi, @gengliangwang . 
Is there a reason not to update another `OrcFilters`? We had better be consistent for both in order to support Hadoop 3 in Spark 3.0.0.
- sql/core/v2.3.4/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFilters.scala

Reply:
Also, cc @wangyum since he is actively working on `Hadoop 3.2` support.